h2/src/docsrc/help/help.csv

```diff
@@ -577,10 +577,13 @@ CREATE ROLE READONLY
 
 "Commands (DDL)","CREATE SCHEMA","
 CREATE SCHEMA [ IF NOT EXISTS ] name [ AUTHORIZATION ownerUserName ]
+[ WITH tableEngineParamName [,...] ]
 ","
 Creates a new schema. If no owner is specified, the current user is used. The
 user that executes the command must have admin rights, as well as the owner.
 Specifying the owner currently has no effect.
+Optional table engine parameters are passed down in the tableEngineParams field
```

Comment:
Good point, fixed this one, will push soon. However, please be informed that docs for CREATE TABLE have the same problem in their current state. Should I fix that as well?